### ISSUE TITLE
Problem: updated metadata from database of record is not synced to the interface

### ISSUE THREAD

**Issue by kellyannewithane** [CCA-Public/dip-access-interface#95]
From user stories (processing archivist, round 2) 

-As a processing Archivist, I want to update the descriptive metadata for a Collection or Folder from the database of record (currently TMS – this might be worth waiting on until CCA has a proper archival management system).

**Comment by jraddaoui** [comment]
Hi @stefanabreitwieser, I'd need more information about the management system, its API specifications and how its metadata should map to this application models.

**Comment by stefanabreitwieser** [comment]
I think @marcbouchercca can answer the more technical aspects of this question for you. In the meantime, I'm putting together a TMS-Archivematica metadata crosswalk and will send it along when it's ready.

**Comment by marcbouchercca** [comment]
Hello, when you say "management system", you mean TMS?

**Comment by jraddaoui** [comment]
Hi @marcbouchercca!

I didn't want to fully asume it's going to be the TMS, from this issue description: "this might be worth waiting on until CCA has a proper archival management system". Also, Justin mentioned that it could be an intermediate application.

**Comment by stefanabreitwieser** [comment]
Yes, TMS -- the time-frame for replacing TMS has been extended since Tim first drafted the plan. We'll definitely need something with TMS compatibility in interim. Thanks!

**Comment by stefanabreitwieser** [comment]
[Archivematica-TMS_crosswalk_final.docx](https://github.com/CCA-Public/dip-access-interface/files/2341010/Archivematica-TMS_crosswalk_final.docx)
And here's the TMS-Archivematica crosswalk on the Collection and Folder levels -- it also maps the TMS fields to their related SQL tables. Let me know if you need any additional information!

**Comment by jraddaoui** [comment]
Thanks @stefanabreitwieser,

That helps to know that there is not too much to map between both applications. But, without knowing something about the API specifications this is still hard to estimate. Considering that such an API exists, that it will be accessible by this application and that a relation can be made between this application resources and the ones from the TMS; as a rough ballpark, this could go between 50 and 100 hours.
